modules/light-clients/08-wasm/types/proposal_handle.go

```diff
@@ -13,31 +15,33 @@ import (
 	"github.com/cosmos/ibc-go/v8/modules/core/exported"
 )
 
-// CheckSubstituteAndUpdateState will try to update the client with the state of the
-// substitute.
-func (cs ClientState) CheckSubstituteAndUpdateState(
-	ctx sdk.Context,
-	_ codec.BinaryCodec,
-	subjectClientStore, substituteClientStore storetypes.KVStore,
-	substituteClient exported.ClientState,
-) error {
+// CheckSubstituteAndUpdateState will verify that a substitute client state is valid and update the subject client state.
+// Note that this method is used only for recovery and will not allow changes to the code hash.
+func (cs ClientState) CheckSubstituteAndUpdateState(ctx sdk.Context, _ codec.BinaryCodec, subjectClientStore, substituteClientStore storetypes.KVStore, substituteClient exported.ClientState) error {
 	var (
 		subjectPrefix    = []byte("subject/")
 		substitutePrefix = []byte("substitute/")
 	)
 
-	_, ok := substituteClient.(*ClientState)
+	substituteClientState, ok := substituteClient.(*ClientState)
 	if !ok {
 		return errorsmod.Wrapf(
 			clienttypes.ErrInvalidClient,
-			fmt.Sprintf("invalid substitute client state. expected type %T, got %T", &ClientState{}, substituteClient),
+			fmt.Sprintf("invalid substitute client state. Expected type %T, got %T", &ClientState{}, substituteClient),
```

Comment:
```suggestion
			fmt.Sprintf("invalid substitute client state: expected type %T, got %T", &ClientState{}, substituteClient),
```

Reply:
fixed!

Reply:
ah, I fixed the unecessary `Sprintf`, not the capitalization. happy merge-conflict Charly

Reply:
i think he just rm the `fmt.Sprintf` but i can add the colon 🤠

Reply:
team :fist_right: :fist_left: work